src/main/webapp/js/instructor.js

```diff
@@ -14,6 +14,11 @@ $(document).ready(function() {
     
     // bind the show picture onhover events
     bindStudentPhotoHoverLink('.profile-pic-icon-hover');
+    
+    /* Formatting searchKey for highlight searchKey plugin ( ['string1','string2',...] )*/
```

Comment:
These happens often in large projects, despite having a unified styleguide - more often than not we get some discrepencies across different teams or files

However, at the minimum - try to stick to the "local convention", use ```//``` instead of ```/*...*/``` for commenting